sporkfish/searcher/searcher_factory.py

```diff
@@ -23,8 +23,12 @@ def _build_moveorder(order_type: MoveOrderMode) -> MoveOrder:
         :rtype: MoveOrder
         :raises TypeError: If the specified order type is not supported.
         """
+        order_type = searcher_cfg.move_order_mode
+        max_depth = searcher_cfg.max_depth
```

Comment:
is `max_depth` only needed for `KILLER_MOVE`? if so suggest move it down to inside the `elif` statement

Reply:
I missed, ty :)